ngraph/core/include/ngraph/function.hpp

```diff
@@ -52,6 +53,16 @@ namespace ngraph
                  const ParameterVector& parameters,
                  const std::string& name = "");
 
+        Function(const ResultVector& results,
+                 const SinkVector& sinks,
+                 const ParameterVector& parameters,
+                 const std::string& name = "");
+
+        Function(const OutputVector& results,
```

Comment:
I think we need to put `OutputVector & sinks` instead of `SinkVector` here to have consistency with results OutputVector.

Reply:
I don't like this idea. OutputVector for results was added to have consistency with existing ctors. If we add OutputVector for sinks, we need to check if we get Sinks or anything else. Also such ctor creates wrong idea for user, that not only Sink children can be sent, but everything.